pkg/libvmi/replicaset.go

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 The KubeVirt Authors.
+ *
+ */
+
+package libvmi
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	v1 "kubevirt.io/api/core/v1"
+)
+
+func NewReplicaSet(vmi *v1.VirtualMachineInstance, replicas int) *v1.VirtualMachineInstanceReplicaSet {
+	replicas32 := int32(replicas)
+	name := "replicaset" + rand.String(5)
```

Comment:
It seems to me you can use the `name` for the `Name` of the rs

Reply:
I'm not sure about that... I cannot say that I understand why `ReplicaSet` (`VMIReplicaSet` after it) has this `selector` field. But since it exists, I've added a dedicated commit to have a different name in this function's code, too.

Reply:
I don't insist